src/components/Item.js

```diff
@@ -42,21 +57,41 @@ const Item = ({ item, handleChange, deleteItem }) => {
   };
 
   return (
-    <li className={`list-item ${addNextPurchaseStyling(item)}`}>
-      <input
-        className="checkbox"
-        type="checkbox"
-        checked={checked}
-        onChange={e => handleChange(e, item)}
-        id={item.id}
-      />
-      <label htmlFor={item.id} className={className}>
-        {item.name} - next purchase in {item.nextPurchase} days
-      </label>
-      <button className="delete" onClick={() => deleteItem(item.id)}>
-        Delete
-      </button>
-    </li>
+    <>
+      <li className={`list-item ${addNextPurchaseStyling(item)}`}>
+        <label htmlFor={item.id}>
+          <input
+            className="checkbox"
+            type="checkbox"
+            checked={checked}
+            onChange={e => handleChange(e, item)}
+            id={item.id}
+          />
+        </label>
+
+        <span className={className} onClick={handleModalChange}>
+          {item.name}
+        </span>
+
+        <button className="delete" onClick={() => deleteItem(item.id)}>
+          Delete
+        </button>
+      </li>
+      {showItemInfo && (
+        <Modal setDisplay={setShowItemInfo}>
```

Comment:
While the feature appears to work fine in the browser, I believe having `<Modal/>` alongside the `<li/>` results in invalid HTML when the modal is open. The modal is a `<div/>` and the only element that is valid to be at the same level as an `<li/>` is another `<li/>`. The fact that it works is the browser being generous to us! 😅 

What I would suggest (though maybe as a refactor later) would be moving `<Modal/>` outside/after the `<ul/>`, though I'm not sure if that will make it more tricky to track which item you're modaling 🤔

Reply:
Hmm, 🤔 about it more, another thing you could try that would be simpler than factoring that out to after the list would be to put it just inside the `</li>`, though I'm not 💯% sure that will work

Reply:
Thank you for pointing this out. This comment totally re-created my respect for `</ul>` and `</li>` elements 😁. 

Moving `Modal` inside the `<li>` element worked perfectly. I tried refactoring too but putting it inside `<li>` saves extra iteration in the List component. Thanks for the advice 🙂.